src/c++/vdo/base/slab-depot.c

```diff
@@ -11,6 +11,7 @@
 #include <linux/log2.h>
 #include <linux/min_heap.h>
 #include <linux/minmax.h>
+#include <linux/version.h>
 
```

Comment:
As in the other file, you need an #ifdef VDO_UPSTREAM around the version.h import.

Reply:
I have added the ifdef